SYCL/BFloat16/bfloat_hw.cpp

```diff
@@ -0,0 +1,60 @@
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: %t.out
```

Comment:
I'm not sure if that is what we're supposed to do, but I honestly don't know.

Maybe someone else from @intel/llvm-reviewers-runtime know.

Reply:
Seems fine to me.